test/passes/translate-to-fuzz_all-features.txt

```diff
@@ -1103,95 +1103,10 @@
      (v128.const i32x4 0x14171109 0x0109440d 0x10031007 0x021d1401)
     )
    )
-   (if
-    (local.tee $1
-     (local.tee $1
-      (block $label$18 (result i32)
-       (call $log-f64
-        (loop $label$19 (result f64)
-         (block
-          (if
-           (i32.eqz
-            (global.get $hangLimit)
-           )
-           (return
-            (tuple.make
-             (i32.const -10)
-             (ref.null any)
-             (ref.null extern)
-            )
-           )
-          )
-          (global.set $hangLimit
-           (i32.sub
-            (global.get $hangLimit)
-            (i32.const 1)
-           )
-          )
-         )
-         (block (result f64)
-          (block $label$20
-           (call $log-f32
-            (block $label$21
-             (call $log-i32
-              (call $hashMemory)
-             )
-             (br $label$19)
-            )
-           )
-           (call $log-i32
-            (call $hashMemory)
-           )
-          )
-          (br_if $label$19
-           (i32.eqz
-            (ref.is_null
-             (ref.null extern)
-            )
-           )
-          )
-          (f64.const 4294967278)
-         )
-        )
-       )
-       (i32.atomic.load16_u offset=3
-        (i32.and
-         (local.get $1)
-         (i32.const 15)
-        )
-       )
-      )
-     )
-    )
-    (block $label$1
-     (return
-      (tuple.make
-       (i32.const -131072)
-       (ref.null any)
-       (ref.null extern)
-      )
-     )
-    )
-    (block $label$2
-     (i32.atomic.store16 offset=4
-      (i32.and
-       (local.tee $1
-        (local.get $1)
-       )
-       (i32.const 15)
-      )
-      (local.tee $1
-       (i32.const 843005738)
-      )
-     )
-     (return
-      (tuple.make
-       (i32.const -8388607)
-       (ref.null any)
-       (ref.null extern)
-      )
-     )
-    )
+   (tuple.make
+    (i32.const 65534)
+    (ref.null any)
+    (ref.null extern)
```

Comment:
Removal of all this code here looks suspicious. Did I miss something?

Reply:
I don't think so. Changing one small thing in fuzzer can have a huge ripple effect and change the output altogether. When you extract `addSingleReferenceTypes` out of `makeRefIsNull`, it looks you changed the order of adding `externref` and `funcref` (which is totally fine), and I guess that's what made this change. Yeah, most PRs that change the fuzzer in anyway have resulted in a huge change in this file, which maybe not great, but anyway I don't think this is a problem.